services/src/pro/users/userdb.rs

```diff
@@ -77,4 +77,29 @@ pub trait UserDb: Send + Sync {
     /// This call fails if the session is invalid
     ///
     async fn set_session_view(&self, session: &UserSession, view: STRectangle) -> Result<()>;
+
+    /// Increments the users quota by the given amount
+    ///
+    /// # Errors
+    ///
+    /// This call fails if the session is invalid
+    ///
+    // TODO: move this method to some AdminDb?
+    async fn increment_quota_used(&self, user: &UserId, quota_used: u64) -> Result<()>;
+
+    /// Gets the current users used quota
+    ///
+    /// # Errors
+    ///
+    /// This call fails if the session is invalid
+    ///
+    async fn quota_used_by_session(&self, session: &UserSession) -> Result<u64>;
+
+    /// Gets a specific users used quota
+    ///
+    /// # Errors
+    ///
+    /// This call fails if the session is invalid
+    ///
+    async fn quota_used_by_user(&self, user: &UserId) -> Result<u64>;
```

Comment:
Do we need both? Or could we derive one from another? E.g. the user id from the session

Reply:
suggest a different name

Reply:
Since `UserSession` contains the field `id: UserId`, we can omit `quota_used_by_session` and call it whenever we have either a `UserId` or a `UserSession`, right?

Reply:
Well I don't know. If your are so hesitant in keeping stuff that we not necessarily need now, because of some admin API that might or might not come, then keep it.

But then at least name it appropriately and update the docs.

Reply:
cf my other comment regardign a QuotaDb

Reply:
Okay, then just rename it

Reply:
rename it to what?

Reply:
I proposed my idea about this 5 days ago.

Reply:
f8a8b3d6f76bad8778c460f2c38f42646f6cb1b8